ansible/roles/controller/tasks/deploy.yml

```diff
@@ -317,6 +317,8 @@
       "CONFIG_whisk_etcd_pool_threads": "{{ etcd.pool_threads }}"
       "CONFIG_whisk_scheduler_grpc_tls": "{{ scheduler.grpc.tls | default('false') | lower }}"
       "CONFIG_whisk_scheduler_maxPeek": "{{ scheduler.maxPeek }}"
+      "CONFIG_whisk_spi_LoadBalancerProvider": "org.apache.openwhisk.core.loadBalancer.FPCPoolBalancer"
```

Comment:
Currently, this is guided to configure them alternatively.
https://github.com/apache/openwhisk/tree/master/ansible#optional-enable-the-new-scheduler

If we take this approach, how about changing the documentation too?

Reply:
this pr is only useful for ansible way(we may need other prs for k8s and standalone), so I think it's better to keep the document